homeworks/kirkovvalentiniván/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,18 @@
+character_name = input("What is your character's name?  ").title().strip()  # Name input
+
+character_age = int(input("How old is your character?  "))                  #Age input
+character_age_days = round(character_age * 365, 0)                          #Age input in days
+
+python_exp = int(input("How many years of experience does he has in python programming? \
```

Comment:
Nem hiba, csak egy javaslat: Ha van egy olyan változód hogy character_age_days, akkor ezt is érdemes úgy elnevezni hogy python_exp_years, így még leíróbb és egységesíti a változóid elnevezését.

Reply:
Szia! Viszont mivel az a változó azt mutatja meg, hogy hány éves napokban és nem a python programozás években kifejezett tapasztalatát (számomra) szerintem leíróbb! Viszont mindenképpen köszi a javaslatot! :)